contents/handbook/growth/marketing/youtube.md

```diff
@@ -0,0 +1,42 @@
+---
+title: YouTube
+sidebar: Handbook
+showTitle: true
+---
+
+YouTube could be a massive channel for organic traffic. Our small amount of [YouTube content (especially tutorials)](https://www.youtube.com/channel/UCoP6ql8QkyOoVpBU4P8LM6w) has a decent amount (x000s) of views. Many potential and current users in our [ICP](/handbook/growth/marketing/customer-personas) rely on YouTube for informative content. Users [have asked for it](https://posthog.slack.com/archives/C02KGGDCA6Q/p1667921203713659).
+
+We have lots of content (blog, tutorial) that can relatively easily turn into video content. As a start, we should turn tutorial and SEO blog content into video content and post it on YouTube. 
+
+What we (Ian) need:
+
+- Something to say (a script or notes)
+- A way to record
+- A way to edit
+- Title, thumbnail, publishing
+
+## Roadmap
+
+v1:
+
+- Ian records using Loom, my equipment, basic tutorials, basically the same script as article, light editing (from me), and templated thumbnails.
+- Goal: consistently publish at least 2 videos per week, with x00s of views each, >0 signups attributable per week.
+
+v2:
+
+- Use Loom, slightly upgrade equipment, more advanced (longer) tutorials, more work on the script, a freelance editor, and custom thumbnails (from Lottie).
+- Goal: consistently publish 2 higher quality videos per week, some reaching x000s  of views, >5 signups attributable per week.
+
+v3:
+
+- Use Loom or other software, same equipment, record blog posts and [YouTube-specific content](https://github.com/PostHog/posthog.com/issues/4586), better scripts,  freelance editor, custom thumbnails (from Lottie or freelancer).
+- Goal: publish 3 high-quality videos per week (at least 1 featuring non-tutorial content), consistently reaching x000s, some videos reaching x0000. >10 signups attributable per week.
+
+## Distribution
+Distributing these videos will consist of:
+- Relying on YouTube's algorithm
+- Adding a link to the video in the related tutorial
```

Comment:
I'd say we should embed, rather than link. Keep people on site, save users click.